src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -71,7 +71,10 @@ private string GetProjectOrSolution()
 
             if (Directory.Exists(resultPath))
             {
-                var possibleSolutionPath = Directory.GetFiles(resultPath, "*.sln", SearchOption.TopDirectoryOnly);
+                string[] possibleSolutionPath = [
+                    ..Directory.GetFiles(resultPath, "*.sln", SearchOption.TopDirectoryOnly),
+                    ..Directory.GetFiles(resultPath, "*.slnx", SearchOption.TopDirectoryOnly)
```

Comment:
This is used so much that I feel like it should be a helper method

Reply:
Agreed, I will look into replacing it with `SlnFileFactory`, thanks!